packit_service/worker/parser.py

```diff
@@ -132,6 +138,50 @@ def parse_pr_event(event) -> Optional[PullRequestEvent]:
             )
         return None
 
+    @staticmethod
+    def parse_pull_request_comment_event(event) -> Optional[PullRequestCommentEvent]:
+        """ look into the provided event and see if it is github issue event """
+        action = nested_get(event, "action")
+        logger.debug(f"action = {action}")
+        is_issue = nested_get(event, "issue")
+        if not is_issue:
+            logger.info("Not a issue event.")
+            return None
+        pr_id = nested_get(event, "issue", "number")
+        is_pr_comment = nested_get(event, "issue", "pull_request")
+        if not is_pr_comment:
+            logger.info("Not a comment on pull request.")
+            return None
+        if action in ["created", "edited"] and pr_id:
```

Comment:
this is definitely a nit

Reply:
I'm not saying that you're wrong, feel free to unresolve and require the changes you propose to be implemented. I just meant to say that I'd personally not make such comment: for me the code looks clear.

We all contribute to this project so @jpopelka if this is important to you, please make the comment blocking the review.

Reply:
If it was blocking the review, I'd make it blocking. Instead, I clicked 'Resolve conversation' - which suggests that I can live with the code as-is.

On the other hand, you're right that we all contribute to this project, so the code should look clear so we all don't have to spend more time (than needed] reading it.
Good for you that it was looking clear to you, but I'm older, slower and not that smart and what's clear to you might not be clear to me.

If I wrote a block of code which might be improved, I'd be definitely glad if you raised that up.
I just meant to say that I'd personally not make such "This is definitely a nit" comment, because if the pull-requester spends less time improving the code than several other people reading it later than it's worth the work (and the comment).

Reply:
The above comment is solely for @TomasTomecek 

Kudos to @phracek for incorporating my suggestions!